src/main/java/org/apache/maven/report/projectinfo/CiManagementReport.java

```diff
@@ -88,6 +91,22 @@ protected String getI18Nsection()
     private static class CiManagementRenderer
         extends AbstractProjectInfoRenderer
     {
+
```

Comment:
I like this change. Can we have a separate PR, preceding this one? I will gladly merge this then.

Reply:
Do you need separate issue for refactor?

Reply:
No, it does not change any behavior. It is just a code cleanup.